controllers/cron_rollout_helpers.go

```diff
@@ -174,41 +175,36 @@ func CreateDisruptionFromTemplate(ctx context.Context, cl client.Client, scheme
 }
 
 // getScheduledTimeForDisruption returns the scheduled time for a particular disruption.
-func getScheduledTimeForDisruption(disruption *chaosv1beta1.Disruption) (*time.Time, error) {
+func getScheduledTimeForDisruption(log *zap.SugaredLogger, disruption *chaosv1beta1.Disruption) time.Time {
 	timeRaw := disruption.Annotations[ScheduledAtAnnotation]
 	if len(timeRaw) == 0 {
-		return nil, nil
+		return time.Time{}
 	}
 
 	timeParsed, err := time.Parse(time.RFC3339, timeRaw)
 	if err != nil {
-		return nil, err
+		log.Errorw("unable to parse schedule time for child disruption", "err", err, "disruption", disruption.Name)
```

Comment:
```suggestion
		log.Errorw("unable to parse schedule time for child disruption", "err", err, "disruptionName", disruption.Name)
```

For consistency with disruption.name tags elsewhere

Reply:
log does not have the right tags already associated? (disruptionCronName, disruptionCronNamespace)